sourcegraph/sourcegraph
# Add telemetry to "Tips and Tricks / More search features" link

Closes https://github.com/sourcegraph/sourcegraph/issues/34516.

## Description
This PR:
- Add telemetry to the "Tips and Tricks / More search features" link
- Add unit tests to the TipsAndTricks component

## Test plan
- `SOURCEGRAPH_API_URL=https://sourcegraph.com SOURCEGRAPHDOTCOM_MODE=true sg start web-standalone`
- Open https://sourcegraph.test:3443/search as non signed-in user
- Enabled event log debug mode in console: `localStorage.setItem('eventLogDebug', 'true')`
- Click "More search features" link and check that `HomepageExampleMoreSearchFeaturesClicked` event is logged in console




## App preview:

- [Web](https://sg-web-erzhtor-add-telemetry-to-tips-and.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-dlakaoatbz.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
